reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -93,14 +93,53 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For anything that executes out-of-runspace, doesn't execute directly in the
+caller's runspace, you need the `Using` scope modifier in order to embed
+variable values from the caller's scope, so that the out-of-runspace code
+can access it. (Conversely, all other contexts neither require nor support the
+'Using' scope modifier.) Specifically, this includes the following contexts:
+
+- Remotely executed commands, started with Invoke-Command's `-ComputerName` or
+  `-Session` parameter (remote runspace)
+- Background jobs, started with Start-Job (out-of-process runspace)
+- Thread jobs, started via Start-ThreadJob (separate runspace)
+- InlineScript in Workflows (out-of-process)
+
+> [!NOTE]
+> Note that out-of-runspace code can never modify variables in the caller's
```

Comment:
Good note.  This is a difficult concept for some users.  I would write it:

'Depending on the context, embedded variables will be either copies of or references to the caller's scope variables.  Out-of-process sessions (remote, background jobs, workflows) embed copies of variables through the PowerShell serialization system (see Serialization of variable values section below).  But for thread jobs [and ForEach-Object -Parallel in 7.0+ versions], references to variables are passed to each thread session.  This means it is possible to modify call scope variables in a different thread, but to do this safely requires thread synchronization.'

Reply:
"reference to variables" is precisely the terminology I suggest we avoid; it's a reference to an _instance of a reference type_, not to the _variable_ - I know that the distinction is a subtle one, but if you phrase it that way, user may expect that passing variable `$foo` with a value-type instance such as `42` can be _assigned to_ via `$using:foo = 43`, which isn't true.

`$using:foo` is the _value_ of a variable, and if that _value_ happens to be a _reference_ (to a mutable object), other threads can modify it.

I know that the value-type / reference-type dichotomy is an advanced topic, but I don't think we should gloss over the distinction here, because it will ultimately cause more confusion.